### PR TITLE
#164198172 Setup DB engine configuration with PostgresSQL

### PR DESCRIPTION
**Pull request by devdbrandy**
#### What does this PR do?

Configure db engine configuration with PostgresSQL

#### Description of Task to be completed

- [x] Install relevant dependencies in conjunction with Sequelize ORM
- [x] Configure sequlize to use PostgresSQL as the database engine
- [x] Setup migration and seeders
- [x] Refactor implementation to meet es6 standard
- [ ] Refactor models to use sequelize pattern of model declaration

#### How should this be manually tested

_Prerequisite: Postgres_

1. Run command:

```bash
git clone -b chore/164198172/configure-postgres --single-branch \
https://github.com/andela/apollo-ah-backend.git
cd apollo-ah-backend
cp .env.example .env
npm install
```

2. Configure `.env` environment variable with your db credentials, then run any of the provided script commands.

- `npm run db:create` - Creates the database.

- `npm run make:model <name> --attributes <attribute>:<type>` - Generates model, where `name` is the name of the model, `attribute` is the attribute of the model and `type` is the attribute data type.

- `npm run migrate`: Runs the migration.

#### Any background context you want to provide

Documentation on how to go about using the cli tool to generate models, migrations, seeders will be provided on the Wiki page.
**[Update]** Documentation is available on [Wiki](https://github.com/andela/apollo-ah-backend/wiki/Guides:-Database-Models,-Migrations-and-Seeding) page.

#### What are the relevant pivotal tracker stories

[#164198172](https://www.pivotaltracker.com/story/show/164198172)

#### Screenshots

n/a

### CODE REVIEW

**Comment by fejiroofficial** [comment]
Can your pivotal link be in this format?
[#164198172](https://www.pivotaltracker.com/n/projects/2313263/stories/164198172)

**Comment by devdbrandy** [comment]
Created wiki page for performing database related operation, link below:
[Guides: Database Models, Migrations and Seeding](https://github.com/andela/apollo-ah-backend/wiki/Guides:-Database-Models,-Migrations-and-Seeding)

File: test/index.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const assert = require('assert');
```
[the remark this answers is not in the document]
Comment (devdbrandy) [reply test/index.test.js]: Hound config setup #4 is dependant on PR #5.

screen shot 2019-02-26 at 1 45 27 pm

**Comment by devdbrandy** [comment]
Hound config setup #4 is dependant on PR #5. Hound checks is based on its default configuration 'esversion: 6'.

screen shot 2019-02-26 at 1 45 27 pm

**Comment by aanchirinah** [comment]
You are not supposed to create a PR for Master, kindly raise it against `Staging`

**Pull request closed without merging**